wrappers/python/src/python/readdy/api/registry/observables.py

```diff
@@ -58,6 +58,7 @@ class Observables(object):
     def __init__(self, simulation):
         self._simulation = simulation
         self._sim = self._simulation._simulation
+        # fixme why keep this in py side, the enable_write_to_file logic can be performed by cpp Simulation
```

Comment:
can be changed I guess. As long as enable_write_to_file is performed on PY side this needs to be stored though

Reply:
yes but out of scope here